src/include/pbs_internal.h

```diff
@@ -480,6 +478,8 @@ extern char *pbs_submit_with_cred(int, struct attropl *, char *,
 
 extern int pbs_loadconf(int);
 
+extern int pbs_query_max_connections(void);
```

Comment:
I have no clue why I moved these, but it's not doing any harm at its new position :) so I'm going to leave these be.